xml/schema/Mailing/Mailing.xml

```diff
@@ -394,6 +394,47 @@
       <formatType>activityDateTime</formatType>
     </html>
   </field>
+  <field>
+    <name>start_date</name>
+    <title>Mailing Start Date</title>
+    <type>timestamp</type>
+    <default>NULL</default>
+    <required>false</required>
+    <comment>When the mailing started to go out</comment>
+    <html>
+      <type>Select Date</type>
+      <formatType>activityDateTime</formatType>
+    </html>
+    <readonly>true</readonly>
+  </field>
+  <field>
+    <name>end_date</name>
+    <title>Mailing End Date</title>
+    <type>timestamp</type>
+    <default>NULL</default>
+    <required>false</required>
+    <comment>When the mailing finished going out.</comment>
+    <html>
+      <type>Select Date</type>
+      <formatType>activityDateTime</formatType>
+    </html>
+    <readonly>true</readonly>
+  </field>
+  <field>
+    <name>status</name>
+    <title>Mailing Status</title>
+    <type>varchar</type>
+    <length>12</length>
+    <comment>The status of this mailing</comment>
+    <default>'Scheduled'</default>
```

Comment:
Shouldn't this be
```suggestion
    <default>'Draft'</default>
```

Reply:
hmm - yeah maybe - seems like a sensile default